sandbox/extension/Humboldt_2023-09-01.xml

```diff
@@ -0,0 +1,87 @@
+    <property group='Site' name='reportedWeather' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/reportedWeather' dc:relation='https://tdwg.github.io/eco/terms/#eco:reportedWeather' dc:description='A list of weather or climatic conditions present during the dwc:Event.' comments='Recommended best practice is to use a key:value encoding schema for a data interchange format such as JSON.' examples='`{&quot;minimumTemperatureInDegreesFahrenheit&quot;: 18, &quot;maximumTemperatureInDegreesFahrenheit&quot;: 32}`' required='false'/>
+    <property group='Site' name='reportedExtremeConditions' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/reportedExtremeConditions' dc:relation='https://tdwg.github.io/eco/terms/#eco:reportedExtremeConditions' dc:description='A description of any extreme weather or environmental conditions that may have affected the dwc:Event.' comments='' examples='`flooding during week 3 of surveys`; `rockslide at site 2`' required='false'/>
+
+    <!-- Habitat Scope -->
+    <property group='Habitat Scope' name='targetHabitatScope' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/targetHabitatScope' dc:relation='https://tdwg.github.io/eco/terms/#eco:targetHabitatScope' dc:description='The habitats targeted for sampling during the dwc:Event.' comments='Recommended best practice is to use a controlled vocabulary and separate the values in a list with space vertical bar space ( | ).' examples='`dunes`; `pine forest`; `riparian`; `scrub | grassland`' required='false'/>
+    <property group='Habitat Scope' name='excludedHabitatScope' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/excludedHabitatScope' dc:relation='https://tdwg.github.io/eco/terms/#eco:excludedHabitatScope' dc:description='The habitats explicitly excluded from sampling during the dwc:Event.' comments='Recommended best practice is to use a controlled vocabulary and separate multiple values in a list with space vertical bar space ( | ).' examples='`wet flatwoods`; `swamp | estuary`' required='false'/>
+
+    <!-- Temporal Scope -->
+    <property group='Temporal Scope' name='eventDuration' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/eventDuration' dc:relation='https://tdwg.github.io/eco/terms/#eco:eventDuration' dc:description='The numeric value for the duration of the dwc:Event.' comments='An eco:eventDuration must have a corresponding eco:eventDurationUnit.' examples='`1`; `30`' required='false'/>
```

Comment:
Would this definition benefit from clear instruction to ensure that any duration provided MUST not contradict any range provided in `eventDate`?